src/Http/Http.Extensions/test/Microsoft.AspNetCore.Http.Extensions.Tests.csproj

```diff
@@ -28,8 +28,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="RequestDelegateGenerator\Baselines\*.generated.txt">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
+    <AssemblyAttribute
+      Include="System.Reflection.AssemblyMetadataAttribute">
+      <_Parameter1>RequestDelegateGeneratorTestBaselines</_Parameter1>
+      <_Parameter2>$([System.IO.Path]::Combine($(MSBuildProjectDirectory), "RequestDelegateGenerator", "Baselines"))</_Parameter2>
```

Comment:
Does this work on things like Helix?

Reply:
Yep, we use this pattern in our E2E tests. There's a little bit more work that I have to do to though get things wired up correctly in the MSBuild, apparently.